spec/lib/zendesk_apps_tools/command_spec.rb

```diff
@@ -144,7 +144,10 @@
         stub_request(:get, PREFIX + '/api/apps.json')
           .with(headers: { 'Authorization' => 'Basic dXNlcm5hbWU6cGFzc3dvcmQ=' })
           .to_return(body: '')
-        expect(@command).to receive(:say_error).with(/^The app was not found./)
+        expect(@command).to receive(:say_error).with(
```

Comment:
Food for thought for test cases like this: do you think it might be worth relaxing the check here, so that if the error message changes the test continues to pass? i.e. How strict should we be in what you would you consider an actual test "failure" here?

(Not at all a blocker btw)

Reply:
I love food (including for thoughts). I now understand how they were using /Regex/ to check for this failure. Thanks for pointing this out.

Before the change, the Error Message was 
`'The app was not found. Please verify your credentials, subdomain, and app name are correct.' ` and they were checking for `/^The app was not found./`. If I wanted to relax the check here it would have been

`"/^App not found./` for the current
`"App not found. Please verify that your credentials, subdomain, and app name are correct."`

I usually prefer to be explicitly declare things for less brain processing. But I am also interested to know what you think? When should I consider relaxing the checks here.